### ISSUE TITLE
CI/CD Pipelines Do Not Generate Schemas for Develop Branch

### ISSUE THREAD

**Issue by aj-stein-nist** [usnistgov/OSCAL#1448]
### User Story

As a NIST or community OSCAL tools developer, in order to quickly test and iterate on model changes in develop before release, I would like model updates _and_ their updated generated schemas to the `develop` branch of this repository.

NOTE: In the 20220908 Lunch with Devs meeting, Dave had claimed this has been working in the past and is likely a regression. I am therefore labeling this a bug for the time being.

Additionally, in terms of branch behavior, `feature-*` branches should behave like `develop` to facilitate developer operations, but they are not included by filter at this time, so team and external contributors cannot really leverage the updated schemas for the feature branch changes.

### Goals

- [x] Support automated schema and docs generation from develop (already working, potential improvement in #1468)
- [ ] Related (maybe stretch goal?) add support for generating schemas and docs from feature branches (new work, trying to meet converted but bug report to AC updates in #1469)

### Dependencies

N/A

### Acceptance Criteria

- [ ] All [OSCAL website](https://pages.nist.gov/OSCAL) and readme documentation affected by the changes in this issue have been updated. Changes to the OSCAL website can be made in the docs/content directory of your branch.
- [ ] A gist is submitted that fully addresses the goals of this User Story. This issue is referenced in the PR.
- [ ] The CI-CD build process runs without any reported errors on the PR. This can be confirmed by reviewing that all checks have passed in the PR.

**Comment by aj-stein-nist** [comment]
Talked with Dave and confirmed that needs to be pulled into sprint and worked.

**Comment by aj-stein-nist** [comment]
Status update: it is working on `develop`, how the commits show has changed.

So once I took a breather and looked at this as asked, I realized the issue is largely cosmetic. We can add back the OSCAL Builder user as the author of the commits, if we choose. That distinguishes them from NIST dev and/or `github-actions[bot]`-tagged commits that blend in with organic development or dependabot-style alerts. This appeared to be removed like this by yours truly sometime around July.

https://github.com/usnistgov/OSCAL/commit/b08851c18c5729e7ea362a9fc121ceffc6b2fa12#diff-cb672281344e580a31f24cc34150f157ab647223ecc517ad7eb5da6b69741d4dL197-L199

Will add a PR as a straw man for review/discussion by Dave.

Having reviewed this issue, I also will clarify the AC with a separate PR to consider enabling this for feature branches as well.

Removing the bug label off of this one.
